library/kotlin/src/io/envoyproxy/envoymobile/Headers.kt

```diff
@@ -0,0 +1,30 @@
+package io.envoyproxy.envoymobile
+
+/*
+ * Base class that is used to represent header/trailer data structures.
+ * To instantiate new instances, see `{Request|Response}HeadersBuilder`.
+ */
+open class Headers {
+  @Suppress("MemberNameEqualsClassName")
```

Comment:
Can we add a TODO for fixing the linter from being this strict?

Reply:
My personal bias is to make it intentional to suppress the linter rather than making the linter less-strict. But I am draconian like that.

This way we don't fall into anti-patterns (according to the linter's convention) without giving it thought.